samples/CDN/create-service.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Copyright 2012-2014 Rackspace US, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+require dirname(__DIR__) . '/../vendor/autoload.php';
+
+use OpenCloud\Rackspace;
+
+// 1. Instantiate a Rackspace client. You can replace {authUrl} with
+// Rackspace::US_IDENTITY_ENDPOINT or similar
+$client = new Rackspace('{authUrl}', array(
+    'username' => '{username}',
+    'apiKey'   => '{apiKey}',
+));
+
+// 2. Obtain an CDN service object from the client.
+$cdnService = $client->cdnService();
+
+// 3. Create service.
+$service = $cdnService->createService(array(
+    'name'     => '{name}',
+    'domains'  => array(
+        array(
+            'domain' => '{domain name}'
```

Comment:
When it comes to short arrays like this, do you have any objection to putting them on 1 line? like `array('domain' => '...')`